how-to-sans-io.rst

```diff
@@ -0,0 +1,328 @@
+Writing I/O-Free (Sans-I/O) Protocol Implementations
+====================================================
+
+This informational document is an attempt to make the case for implementing
+network protocols without writing I/O, and to provide concrete assistance and
```

Comment:
"writing" -> it's actually reading and writing, so maybe say "implementing any", "using any", or "writing any I/O code"?